boms/cloud-lts-bom/pom.xml

```diff
@@ -299,13 +298,11 @@
       </dependency>
       <dependency>
         <groupId>com.google.cloud</groupId>
-        <!-- BOM? -->
         <artifactId>google-cloud-trace-bom</artifactId>
```

Comment:
@Neenu1995 Can I use this BOM to cover API surface?

Reply:
yes